File: Ass4/ass4src/c++/XML employee example/Employee.cpp
#include "Employee.h"
#include <iostream>

Employee::Employee(void)
{
	fName = "";
	fAge = 0;
	fId = 0;
	fType = "";
}

Employee::Employee(std::string name, int age, int id, std::string type):fName(name), fAge(age), fId(id), fType(type)
{
}

Employee::~Employee(void)
{
}

std::string Employee::toString()
{
	std::stringstream ss;
	ss << "Employee\n\tName:" << fName << "\n\tAge:" << fAge << "\n\tId:" << fId << "\n\tType:" << fType << "\n"; 
	return ss.str();
}

File: Ass4/ass4src/java/xmlParser/src/dataStructure/Employee.java
package dataStructure;

public class Employee {
	private String fName;
	private int fAge;
	private int fId;
	private String fType;
	
	public Employee(String name, int age, int id, String type){
		fName = name;
		fAge = age;
		fId = id;
		fType = type;
	}
	
	public String toString(){
		return "Employee\n\tName:" + fName + "\n\tAge:" + fAge + "\n\tId:" + fId + "\n\tType:" + fType + "\n"; 
	}
	
}
